xml/System.Collections.Immutable/ImmutableArray`1+Builder.xml

```diff
@@ -773,36 +773,6 @@
         <exception cref="T:System.IndexOutOfRangeException">The specified index is not in the array.</exception>
       </Docs>
     </Member>
-    <Member MemberName="ItemRef">
```

Comment:
@dend @joelmartinez is there a new version of mdoc? I wasn't expecting any diffs like this.